internal/services/environment/resource_environment_test.go

```diff
@@ -278,6 +278,93 @@ func TestUnitEnvironmentsResource_Validate_CreateDeveloperEnvironment(t *testing
 					resource.TestCheckResourceAttr("powerplatform_environment.development", "dataverse.version", "9.2.23092.00206"),
 					resource.TestCheckResourceAttr("powerplatform_environment.development", "dataverse.unique_name", "00000000-0000-0000-0000-000000000001"),
 					resource.TestCheckResourceAttr("powerplatform_environment.development", "billing_policy_id", ""),
+					resource.TestCheckResourceAttr("powerplatform_environment.development", "release_cycle", "Standard"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccEnvironmentsResource_Validate_Create_Early_Release_Cycle(t *testing.T) {
+	t.Setenv("TF_ACC", "1")
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: mocks.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+				resource "powerplatform_environment" "development" {
+					display_name                              = "` + mocks.TestName() + `"
+					location                                  = "unitedstates"
+					environment_type                          = "Sandbox"
```

Comment:
ensure you test with all environment types ... if it doesn't work with one we can document that as expected